connectomics/utils/processing/process_syn.py

```diff
@@ -82,4 +82,14 @@ def polarity2instance(volume, thres=0.5, thres_small=128,
                        int(segm.shape[1]*scale_factors[1]), 
                        int(segm.shape[2]*scale_factors[2]))
         segm = resize(segm, target_size, order=0, anti_aliasing=False, preserve_range=True)
-    return segm
\ No newline at end of file
+    return segm
+
+def main():
+    vol_path = sys.argv[1]
+    format = sys.argv[2]

```

Comment:
Hope these 2 arguments are fine. 
1) vol_path refers to the predicted volume
2) format is for the saving function. H5/PNG.

Usage: `python process_syn.py path/to/file png`